package.json

```diff
@@ -12,15 +12,15 @@
     "test:e2e:ci": "vue-cli-service test:e2e --headless"
   },
   "dependencies": {
-    "@mdi/font": "^4.7.95",
-    "buefy": "^0.8.8",
-    "core-js": "^3.6.1",
+    "@mdi/font": "^4.9.95",
+    "buefy": "^0.8.10",
+    "core-js": "^3.6.4",
     "svg-pan-zoom": "^3.6.1",
     "vue": "^2.6.11",
     "vuex": "^3.1.2"
   },
   "devDependencies": {
-    "@types/jest": "^24.0.25",
+    "@types/jest": "^24.9.0",
```

Comment:
[Newest jest is 25.1.0](https://github.com/facebook/jest/releases)

Reply:
Discussed on Slack. Some dependencies rely on the Vue CLI plugins. Jest is one of these dependencies that we can't update to the latest yet.